configure

```diff
@@ -814,6 +814,23 @@ def host_arch_win():
   return matchup.get(arch, 'ia32')
 
 
+def libc():
+  try:
+    proc = subprocess.Popen(
+        ['ldd', '--version'],
```

Comment:
`ldd --version` is not a reliable indicator of whether the node binary is going to be linked against glibc or musl.

It could work if you ran `ldd out/Release/node` post-build but that doesn't help you here, of course.  I suppose you could `dlsym("some_symbol_that_only_exists_in_musl_or_glibc") != nullptr` at run-time.